helper-cli/src/funTest/assets/package-list.yml

```diff
@@ -0,0 +1,23 @@
+---
+name: "Example project name"
+vcs_type: "Git"
+vcs_url: "https://github.com/example/project.git"
+vcs_revision: "2222222222222222222222222222222222222222"
+vcs_path: "vcs-path/project"
+dependencies:
+  - id: "NPM::example-dependency-one:1.0.0"
+    vcs_type: "Git"
+    vcs_url: "https://github.com/example/depedency-one.git"
+    vcs_revision: "0000000000000000000000000000000000000000"
+    vcs_path: "vcs-path/dependency-one"
+    source_artifact_url: "https://example.org/example-dependency-one.zip"
+    is_excluded: true
+    is_dynamically_linked: true
```

Comment:
This command targets the use case of doing a quick (as possible) scan and clearance for an arbitrary set of packages. In that use case it is unneeded. I also do not foresee to have subprojects. 

From our discussion in the developer meeting, what I understood was, (from @mnonnenmacher) there is need for an analyzer which is a bit simpler than SPDX. 
And that he preferred to have my simple use case be a helper-cli command, so that this simple use case would conflict with @mnonnenmacher analyzer use case. So, given that I prefer we keep this helper-command to a minimum and add the more complex cases to @mnonnenmacher 's analyzer use case if it fits there?!